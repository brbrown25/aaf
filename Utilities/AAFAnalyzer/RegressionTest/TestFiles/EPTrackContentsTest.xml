<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE aaf-file SYSTEM "EPDerivationChain.dtd">

<aaf-file>

    <!-- Test Sub-Clip Compositions -->

	<sub-clip name="Sub-Clip - Audio Timeline Source Clip">
        <source-clip slot-type="timeline" track-type="audio" edit-rate-numerator="1" edit-rate-denominator="1">
			<template-clip name="Template Clip A"><eoc/></template-clip>
        </source-clip>
    </sub-clip>  
    <sub-clip name="Sub-Clip - Audio Static Source Clip">
        <source-clip slot-type="static" track-type="audio" edit-rate-numerator="1" edit-rate-denominator="1">
			<template-clip name="Template Clip B"><eoc/></template-clip>
        </source-clip>
    </sub-clip>   
    <sub-clip name="Sub-Clip - Video Timeline Source Clip">
        <source-clip slot-type="timeline" track-type="picture" edit-rate-numerator="1" edit-rate-denominator="1">
			<template-clip name="Template Clip D"><eoc/></template-clip>
        </source-clip>
    </sub-clip>
    <sub-clip name="Sub-Clip - Video Static Source Clip">
        <source-clip slot-type="static" track-type="picture" edit-rate-numerator="1" edit-rate-denominator="1">
			<template-clip name="Template Clip E"><eoc/></template-clip>
        </source-clip>
    </sub-clip>

    <sub-clip name="Sub-Clip - Audio Timeline Operation Group">
        <operation-group slot-type="timeline" track-type="audio" edit-rate-numerator="1" edit-rate-denominator="1">
			<template-clip name="Template Clip G"><eoc/></template-clip>
        </operation-group>
    </sub-clip>
    <sub-clip name="Sub-Clip - Audio Static Operation Group">
        <operation-group slot-type="static" track-type="audio" edit-rate-numerator="1" edit-rate-denominator="1">
            <template-clip name="Template Clip H"><eoc/></template-clip>
        </operation-group>
    </sub-clip>
    <sub-clip name="Sub-Clip - Video Timeline Operation Group">
        <operation-group slot-type="timeline" track-type="picture" edit-rate-numerator="1" edit-rate-denominator="1">
            <template-clip name="Template Clip J"><eoc/></template-clip>
        </operation-group>
    </sub-clip>
    <sub-clip name="Sub-Clip - Video Static Operation Group">
        <operation-group slot-type="static" track-type="picture" edit-rate-numerator="1" edit-rate-denominator="1">
            <template-clip name="Template Clip K"><eoc/></template-clip>
        </operation-group>
    </sub-clip>

    <sub-clip name="Sub-Clip - Timecode Track">
        <timecode/>
        <eoc/>
    </sub-clip>
    
    <!-- Test Adjusted-Clip Compositions -->

	<adjusted-clip name="Adjusted Clip - Audio Timeline Source Clip">
        <source-clip slot-type="timeline" track-type="audio" edit-rate-numerator="1" edit-rate-denominator="1">
			<template-clip name="Template Clip M"><eoc/></template-clip>
        </source-clip>
    </adjusted-clip>
    <adjusted-clip name="Adjusted Clip - Audio Static Source Clip">
        <source-clip slot-type="static" track-type="audio" edit-rate-numerator="1" edit-rate-denominator="1">
			<template-clip name="Template Clip N"><eoc/></template-clip>
        </source-clip>
    </adjusted-clip>
    <adjusted-clip name="Adjusted Clip - Video Timeline Source Clip">
        <source-clip slot-type="timeline" track-type="picture" edit-rate-numerator="1" edit-rate-denominator="1">
			<template-clip name="Template Clip P"><eoc/></template-clip>
        </source-clip>
    </adjusted-clip>
    <adjusted-clip name="Adjusted Clip - Video Static Source Clip">
        <source-clip slot-type="static" track-type="picture" edit-rate-numerator="1" edit-rate-denominator="1">
			<template-clip name="Template Clip Q"><eoc/></template-clip>
        </source-clip>
    </adjusted-clip>

    <adjusted-clip name="Adjusted Clip - Audio Timeline Operation Group">
        <operation-group slot-type="timeline" track-type="audio" edit-rate-numerator="1" edit-rate-denominator="1">
			<template-clip name="Template Clip S"><eoc/></template-clip>
        </operation-group>
    </adjusted-clip>
    <adjusted-clip name="Adjusted Clip - Audio Static Operation Group">
        <operation-group slot-type="static" track-type="audio" edit-rate-numerator="1" edit-rate-denominator="1">
            <template-clip name="Template Clip T"><eoc/></template-clip>
        </operation-group>
    </adjusted-clip>
    <adjusted-clip name="Adjusted Clip - Video Timeline Operation Group">
        <operation-group slot-type="timeline" track-type="picture" edit-rate-numerator="1" edit-rate-denominator="1">
            <template-clip name="Template Clip V"><eoc/></template-clip>
        </operation-group>
    </adjusted-clip>
    <adjusted-clip name="Adjusted Clip - Video Static Operation Group">
        <operation-group slot-type="static" track-type="picture" edit-rate-numerator="1" edit-rate-denominator="1">
            <template-clip name="Template Clip W"><eoc/></template-clip>
        </operation-group>
    </adjusted-clip>

    <adjusted-clip name="Adjusted Clip - Timecode Track">
        <timecode/>
        <eoc/>
    </adjusted-clip>
    
</aaf-file>