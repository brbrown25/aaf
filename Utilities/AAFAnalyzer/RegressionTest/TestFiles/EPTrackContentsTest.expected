        Name:   Edit Protocol Track Contents Test
        Desc:   Verify the correctness of data within the tracks of Edit Protocol mobs.
        Result: Failed
        Reason: Test Failed - See "Edit Protocol Track Contents Visitor" Visitor for details
        Passing Requirements: None
        Warning Requirements: None
        Failing Requirements: REQ_EP_103

            Name:   Edit Protocol Track Contents Visitor
            Desc:   Visit derivation chain material and make sure that their tracks contain the necessary data.
            Result: Failed
            Reason: 
            Passing Requirements: None
            Warning Requirements: None
            Failing Requirements: REQ_EP_103

                Name:   Edit Protocol Track Contents Visitor
                Desc:   -
                Result: Failed
                Reason: Slot with ID 1 in Top-Level Composition: "Top-Level Essence Without Physical Track Number" does not have a MobSlot::PhysicalTrackNumber property.
                Passing Requirements: None
                Warning Requirements: None
                Failing Requirements: REQ_EP_103

                Name:   Edit Protocol Track Contents Visitor
                Desc:   -
                Result: Failed
                Reason: Slot with ID 1 in Lower-Level Composition: "Lower-Level Essence Without Physical Track Number" does not have a MobSlot::PhysicalTrackNumber property.
                Passing Requirements: None
                Warning Requirements: None
                Failing Requirements: REQ_EP_103

                Name:   Edit Protocol Track Contents Visitor
                Desc:   -
                Result: Failed
                Reason: Slot with ID 1 in Adjusted-Clip Composition: "Adjusted Clip Essence Without Physical Track Number" does not have a MobSlot::PhysicalTrackNumber property.
                Passing Requirements: None
                Warning Requirements: None
                Failing Requirements: REQ_EP_103

                Name:   Edit Protocol Track Contents Visitor
                Desc:   -
                Result: Failed
                Reason: Slot with ID 1 in Sub-Clip Composition: "Sub-Clip Essence Without Physical Track Number" does not have a MobSlot::PhysicalTrackNumber property.
                Passing Requirements: None
                Warning Requirements: None
                Failing Requirements: REQ_EP_103

                Name:   Edit Protocol Track Contents Visitor
                Desc:   -
                Result: Failed
                Reason: Slot with ID 1 in Template Clip: "Template Essence Without Physical Track Number" does not have a MobSlot::PhysicalTrackNumber property.
                Passing Requirements: None
                Warning Requirements: None
                Failing Requirements: REQ_EP_103

                Name:   Edit Protocol Track Contents Visitor
                Desc:   -
                Result: Failed
                Reason: Slot with ID 1 in Clip: "Clip Essence Without Physical Track Number" does not have a MobSlot::PhysicalTrackNumber property.
                Passing Requirements: None
                Warning Requirements: None
                Failing Requirements: REQ_EP_103

                Name:   Edit Protocol Track Contents Visitor
                Desc:   -
                Result: Failed
                Reason: Slot with ID 1 in Recording Source: "Recording Source Essence Without Physical Track Number" does not have a MobSlot::PhysicalTrackNumber property.
                Passing Requirements: None
                Warning Requirements: None
                Failing Requirements: REQ_EP_103

                Name:   Edit Protocol Track Contents Visitor
                Desc:   -
                Result: Failed
                Reason: Slot with ID 1 in Import Source: "Import Source Essence Without Physical Track Number" does not have a MobSlot::PhysicalTrackNumber property.
                Passing Requirements: None
                Warning Requirements: None
                Failing Requirements: REQ_EP_103

                Name:   Edit Protocol Track Contents Visitor
                Desc:   -
                Result: Failed
                Reason: Slot with ID 1 in Tape Source: "Tape Source Essence Without Physical Track Number" does not have a MobSlot::PhysicalTrackNumber property.
                Passing Requirements: None
                Warning Requirements: None
                Failing Requirements: REQ_EP_103

                Name:   Edit Protocol Track Contents Visitor
                Desc:   -
                Result: Failed
                Reason: Slot with ID 1 in Film Source: "Film Source Essence Without Physical Track Number" does not have a MobSlot::PhysicalTrackNumber property.
                Passing Requirements: None
                Warning Requirements: None
                Failing Requirements: REQ_EP_103

